### ISSUE TITLE
Market select shouldn't show when there#s only one market.

### ISSUE THREAD

**Issue by sakulstra** [aave/interface#641]
## Prerequisites

- [x] I have searched the [issues](https://github.com/aave/aave-ui/issues) of this repository and believe that this is not a duplicate.

## Current Behavior 😯

When there's only one market, there's stil the market select dropdown - which has only one choice.
![image](https://user-images.githubusercontent.com/4396533/139849381-9daebb47-130f-4ffa-a3a7-417c14d4c26b.png)

## Expected Behavior 🤔

Dropdown should disappear when it doesn't make sense?

## Steps to Reproduce 🕹

Disable all markets except one.

**Comment by sakulstra** [comment]
@iamanastasia yes still relevant in case arc/rwa upgrade to this version, but the screens are outdated now.
I think it's essentially just about not showing a dropdown arrow when there's only one choice.

**Comment by foodaka** [comment]
i think we can close this out? closing unless we still ahve this issue on v3
